wvm-apps/wvm-exexed/crates/reth-exexed/src/util.rs

```diff
@@ -0,0 +1,40 @@
+use precompiles::inner::graphql_util::send_graphql;
+
+pub const AR_GRAPHQL_GATEWAY: &str = "https://arweave.mainnet.irys.xyz";
+pub const WEVM_OWNER_WALLET: &str = "5JUE58yemNynRDeQDyVECKbGVCQbnX7unPrBRqCPVn5Z";
```

Comment:
WVM :)

Reply:
also instead of having a single owner for now, maybe lets make the GQL query more dynamic by having owners as an array? that way we can add owners in the future more dynamically